packages/form-js-viewer/assets/form-js-base.css

```diff
@@ -75,7 +75,7 @@
     --cds-border-subtle, 
     var(--cds-border-subtle-01, var(--color-grey-225-10-85))
   );
-  --color-borders-readonly: var(--cds-border-disabled, var(--color-grey-225-10-75));
+  --color-borders-readonly: var(--cds-border-subtle, var(--color-grey-225-10-75));
```

Comment:
[The border color was wrong ](https://react.carbondesignsystem.com/?path=/story/components-select--playground&args=readOnly:true)